index.html

```diff
@@ -166,5 +166,30 @@ <h3 class="skills"><img src="assets/Rectangle-3.png" alt="">Skills</h3>
             </section>
         </div>
     </div>
+    <section class="contact">
+        <div>
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form class="contact-form" method="post">
+                <input type="text" placeholder="Full name" id="name" name="user_name">
+                <input type="email" placeholder="Email Address" id="mail" name="user_email">
+                <textarea id="msg" placeholder="Enter Text Here" name="user_message"></textarea>
+                <button type="submit">Get in Touch</button>
```

Comment:
- [ ] Kindly add a hover effect on your button it's currently missing, kindy follow the Figma pattern when implementing it.
![image](https://user-images.githubusercontent.com/61408860/178569824-74d3a0e2-9325-4307-b817-2b41a5828a09.png)

Reply:
Kindly check my css code i added hover to the button and it is working in my browser.

Reply:
@danmainah Kindly check my CSS code I already added the hover effect to the button and it is working in my browser.

Reply:
@danmainah kindly check the class `.contact-form button:hover in CSS file for hover button``